rest-server/engines/src/main/scala/com/actionml/engines/ur/URPreparator.scala

```diff
@@ -36,56 +36,27 @@ object URPreparator extends LazyLogging {
     eventsRDD: RDD[UREvent],
     itemsRDD: RDD[URItemProperties])(implicit sc: SparkContext): TrainingData = {
 
-    //val indicatorParams = dsp.indicatorParams
-
-    // beware! the following call most likely will alter the event stream in the DB!
-    //cleanPersistedPEvents(sc) // broken in apache-pio v0.10.0-incubating it erases all data!!!!!!
-
-    /*
-    val eventsRDD = PEventStore.find(
-      appName = dsp.appName,
-      entityType = Some("user"),
-      indicatorParams = Some(indicatorParams),
-      targetEntityType = Some(Some("item")))(sc).repartition(sc.defaultParallelism)
-    */
-
     // now separate the events by event name
     val indicatorNames = indicatorParams.map(_.name)
 
     logger.info(s"Indicators: ${indicatorNames}")
 
-    //val collectedEventsRDD = eventsRDD.collect()
+    //val debug = eventsRDD.collect()
```

Comment:
^

Reply:
@andreyshikov when you `.collect` any Spark data structure like an RDD it brings it all into memory in one process. This is ok for some debuging but very seldom used in real production Spark code. I have a convention of naming any collected thing "debug" so I can tell to not include them in production code. I would uncomment when using IntelliJ with small data. I have accidentally left these in when pushing so now I can just search for "debug" before pushing to make sure they are all taken out.